tracer/test/Datadog.Trace.TestHelpers/RemoteConfigTestHelper.cs

```diff
@@ -30,7 +30,7 @@ internal static GetRcmResponse SetupRcm(this MockTracerAgent agent, ITestOutputH
 
         internal static async Task<GetRcmRequest> SetupRcmAndWait(this MockTracerAgent agent, ITestOutputHelper output, IEnumerable<(object Config, string ProductName, string Id)> configurations, int timeoutInMilliseconds = WaitForAcknowledgmentTimeout)
         {
-            var response = BuildRcmResponse(configurations.Select(c => (JsonConvert.SerializeObject(c.Config), c.ProductName, c.Id)));
+            var response = BuildRcmResponse(configurations.Select(c => (JsonConvert.SerializeObject(c.Config, new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }), c.ProductName, c.Id)));
```

Comment:
Is this actually what the _real_ RCM responses look like? 🤔 i.e. do they include `null` values rather than missing values or not?

Reply:
Native WAF was refusing to load payloads with null fields,  so...